src/libraries/System.Net.Quic/tests/FunctionalTests/QuicConnectionTests.cs

```diff
@@ -285,6 +285,7 @@ public async Task Dispose_WithOpenLocalStream_LocalStreamFailsWithQuicOperationA
         }
     }
 
+    [ConditionalClass(typeof(QuicTestBase<MockProviderFactory>), nameof(QuicTestBase<MockProviderFactory>.IsSupported))]
```

Comment:
As far as I understand it, we build the S.N.Quic tests only for Linux and Windowsm so they are never run on any of the platforms where we throw PNSE from everything. Thus, the condition was never needed.
But I do agree it doesn't hurt here, and at least it won't break if/when we expand the target OS list.